Dockerfile.in

```diff
@@ -14,7 +14,7 @@
 
 FROM ARG_FROM
 
-MAINTAINER Jimmi Dyson <jimmidyson@gmail.com>
+MAINTAINER Roland Huss <rhuss@redhat.com>
```

Comment:
Cheeky, perhaps dev@syndesis.io

Reply:
Damned, you got me while spreading breadcrumbs.

Changed this in the other PR, not sure whether this PR is relevant anymore.